usecases/config/environment.go

```diff
@@ -239,7 +239,7 @@ func FromEnv(config *Config) error {
 			config.Authorization.Rbac.RootGroups = strings.Split(groupString, ",")
 		}
 
-		viewerGroupString, ok := os.LookupEnv("AUTHORIZATION_RBAC_READONLY_ROOT_GROUPS")
+		viewerGroupString, ok := os.LookupEnv("EXPERIMENTAL_AUTHORIZATION_RBAC_READONLY_ROOT_GROUPS")
```

Comment:
Do we want to add the experimental string also [here](https://github.com/weaviate/weaviate/blob/stable/v1.28/usecases/auth/authorization/rbac/rbacconf/config.go#L22)?